src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -100,4 +100,11 @@ function Base.show(io::IO, ::MIME"text/plain", limiter::SubcellLimiterIDP)
         summary_box(io, "SubcellLimiterIDP", setup)
     end
 end
+
+function get_node_variables!(node_variables, limiter::SubcellLimiterIDP,
+                             ::VolumeIntegralSubcellLimiting, equations)
+    node_variables[:alpha_limiter] = limiter.cache.subcell_limiter_coefficients.alpha
```

Comment:
Is there a more descriptive name than "alpha" possible? If you can't think of anything better immediately, just leave it as it is though 🙂

Reply:
I'll change it to `limiting_coefficient`.